griptape/tools/griptape_cloud_tool/tool.py

```diff
@@ -49,26 +53,31 @@ def _get_schema(self) -> dict:
     def _parse_schema(self, schema: dict) -> tuple[str, dict[str, tuple[str, Schema]]]:
         """Parses an openapi schema into a dictionary of activity names and their respective descriptions + schemas."""
         activities = {}
-
-        name = schema.get("info", {}).get("title")
-
-        for path, path_info in schema.get("paths", {}).items():
-            if not path.startswith("/activities"):
-                continue
-            for method, method_info in path_info.items():
-                if "post" in method.lower():
-                    activity_name = method_info["operationId"]
-                    description = method_info.get("description", "")
-
-                    activity_schema = self.__extract_schema_from_ref(
-                        schema,
-                        method_info.get("requestBody", {})
-                        .get("content", {})
-                        .get("application/json", {})
-                        .get("schema", {}),
-                    )
-
-                    activities[activity_name] = (description, activity_schema)
+        name = ""
+        try:
+            name = schema.get("info", {}).get("title")
+
+            for path, path_info in schema.get("paths", {}).items():
+                if not path.startswith("/activities"):
+                    continue
+                for method, method_info in path_info.items():
+                    if "post" in method.lower():
+                        activity_name = method_info["operationId"]
+                        description = method_info.get("description", "")
+
+                        activity_schema = self.__extract_schema_from_ref(
+                            schema,
+                            method_info.get("requestBody", {})
+                            .get("content", {})
+                            .get("application/json", {})
+                            .get("schema", {}),
+                        )
+
+                        activities[activity_name] = (description, activity_schema)
+
+        except Exception as e:
```

Comment:
Is there anything more specific we can catch here?

Reply:
going to rework this, first pass at an issue we found.

Reply:
hopefully this addresses the root cause